README.md

```diff
@@ -4,37 +4,105 @@
 
 The main objective of the Lab Results Adaptor is to hide complex legacy standards and instead present a simple and consistent interface aligned to current NHSD national standards. The adaptor removes the requirement for a GP System to handle the complexities of EDIFACT and MESH messaging.
 
-At a high level, the Lab Results Adaptor exposes a queue from which the GP System can consume medical reports
+At a high level, the Lab Results Adaptor exposes a queue (AMQP 1.0) from which the GP System can consume Pathology and Screening medical reports in FHIR DSTU3 format.
```

Comment:
Why is this version written as "DSTU3"? I see on http://hl7.org/fhir/stu3/ the version is written as "STU 3". Also should we be more clear and say `FHIR v3.0.2 (STU 3) format`?